examples/lb3-application/public/lb3-index.html

```diff
@@ -0,0 +1,5 @@
+<head><title>LoopBack</title></head>
+<body>
+    <h1>LoopBack Rocks!</h1>
+    <p>Hello World... </p>
```

Comment:
I am concerned about discoverability of this page - how can app users learn about the existence of `/lb3-index.html` URL? Can you perhaps add a link to this page to the main `index.html`? For example:

```html
    <h3>OpenAPI spec: <a href="/openapi.json">/openapi.json</a></h3>
    <h3>API Explorer: <a href="/explorer">/explorer</a></h3>
    <h3><a href="/lb3-index.html">Home page from the LB3 app</a></h3>
```